vendor/venv-update

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
venv-update is python3-compatible, I thought? since this script is vendored, perhaps we should update it to the latest (3.1.1, from 3.0.0)?

Reply:
Yes, that makes sense. I think though, that I would prefer to do that in a separate PR, I don't want to do too much in one change set.

Reply:
sgtm

Reply:
The [latest still uses python2](https://github.com/Yelp/venv-update/blob/master/venv_update.py), but I don't think we really need to worry about this, since it's just a bootstrapping script meant to get dependencies and a virtualenv up and running, not one of our scripts, so it's always meant to use the system version of python 2 without any dependencies.

Reply:
@jvperrin, does it? I don't see anything indicating it won't work with Python 3. Perhaps @asottile can say with certainty?

Reply:
venv-update works fine on python 3

Reply:
Good to know. Thanks :)